src/components/profile/Recovery/RecoveryContainer.js

```diff
@@ -43,7 +43,7 @@ const RecoveryContainer = ({ type, recoveryMethods }) => {
     const currentActiveKinds = new Set(activeMethods.map(method => method.kind));
     const missingKinds = allKinds.filter(kind => !currentActiveKinds.has(kind))
     const deleteAllowed = [...currentActiveKinds].length > 1 || account.ledgerKey;
-    const recoveryLoader = (actionsPending('LOAD_RECOVERY_METHODS') && !userRecoveryMethods.length) || !account.accountId
+    const recoveryLoader = (actionsPending('LOAD_RECOVERY_METHODS') && !userRecoveryMethods.length) || !account.accountId || !account.balance.stakedBalanceMainAccount
```

Comment:
not clear why recovery loader is tied to `account.balance.stakedBalanceMainAccount`

Reply:
before it was showing boxes even if the values are not loaded - now it will show skeleton until the full balance is loaded (CH is working)